packages/r/rendergraph/xmake.lua

```diff
@@ -0,0 +1,42 @@
+package("rendergraph")
+
+    set_homepage("https://github.com/DragonJoker/RenderGraph/")
+    set_description("Vulkan render graph management library. .")
+
+    set_urls("https://github.com/DragonJoker/RenderGraph.git")
+    add_versions("1.0", "61e814bb0298983eae853d9ba5386a272ebc1eb9")
+
+    add_deps("vulkan-headers")
+
+    add_links("RenderGraph")
+
+    on_install("windows|x64", "macosx", "linux", function (package)
+        io.writefile("xmake.lua", [[
+            add_rules("mode.debug", "mode.release")
+            add_requires("vulkan-headers")
+            target("RenderGraph")
+                set_kind("shared")
+                add_includedirs("include")
+                add_files("source/RenderGraph/RunnablePasses/*.cpp")
+                add_files("source/RenderGraph/*.cpp")
+                set_languages("c++20")
+                add_defines("RenderGraph_EXPORTS")
+                add_headerfiles("include/RenderGraph/RunnablePasses/*.hpp", {prefixdir="RenderGraph/RunnablePasses"})
+                add_headerfiles("include/RenderGraph/*.hpp", {prefixdir="RenderGraph"})
+                add_packages("vulkan-headers")
+        ]])
```

Comment:
add configs to switch static/shared kind. see https://github.com/xmake-io/xmake-repo/blob/f574cfe3aeb21fa157155b907a5a037b17f5ff5e/packages/l/libpng/xmake.lua#L48-L49

Reply:
So, in package installation :
```
if package:config("shared") then 
    configs.kind = "shared"
end
```
In the generated xmake.lua :
```
set_kind("$(kind)")
```
Right ?

Reply:
right